app/javascript/actionsPanel/initializeActionsPanelToggle.js

```diff
@@ -1,7 +1,7 @@
 /** This initializes the mod actions button on the article show page (app/views/articles/show.html.erb). */
 export default function initializeActionsPanel(user, path) {
   const modActionsMenuHTML = `
-    <iframe id="mod-container" src=${path}/actions_panel title="Moderation panel actions">
+    <iframe id="mod-container" src=${path}/actions_panel sandbox="allow-top-navigation allow-scripts allow-same-origin allow-modals" title="Moderation panel actions">
```

Comment:
I'm not sure I see the connection between the Pusher error and this sandboxed iframe. BTW, MDN strongly discourages using both `allow-script` and `allow-same-origin` as the script can remove the sandboxing:

> When the embedded document has the same origin as the embedding page, it is strongly discouraged to use both allow-scripts and allow-same-origin, as that lets the embedded document remove the sandbox attribute — making it no more secure than not using the sandbox attribute at all.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe

If the goal is the to redirect the parent page, I was able to make this work without `sandbox`...